src/backend/aspen/workflows/nextstrain_run/run_nextstrain_ondemand.sh

```diff
@@ -17,7 +17,7 @@ start_time=$(date +%s)
 
 aws configure set region $AWS_REGION
 
-if [ ! -z "${BOTO_ENDPOINT_URL}" ]; then
+if [ -n "${BOTO_ENDPOINT_URL-}" ]; then
   export aws="aws --endpoint-url ${BOTO_ENDPOINT_URL}"
```

Comment:
what's the difference between `${BOTO_ENDPOINT_URL-}` in line 20 and `${BOTO_ENDPOINT_URL}` in line 21?

Reply:
The trailing `-` means the script won't exit with an error if that variable doesn't exist.